jcl/src/jdk.jcmd/share/classes/openj9/tools/attach/diagnostics/attacher/AttacherDiagnosticsProvider.java

```diff
@@ -65,6 +65,22 @@ public DiagnosticsInfo getThreadGroupInfo(boolean printSynchronizers) throws IOE
 		return info;
 	}
 
+	/**
+	 * Request thread information, including stack traces, from a target VM.
+	 * 
+	 * @param diagnosticCommand name of command to execute
+	 * @return properties object containing serialized thread information
+	 * @throws IOException in case of a communication error
+	 */
+	public Properties executeDiagnosticCommand(String diagnosticCommand) throws IOException {
+		IPC.logMessage("enter executeDiagnosticCommand ", diagnosticCommand); //$NON-NLS-1$
```

Comment:
unwanted space

Reply:
`logMessage(String string1, String string2)` directly concatenates `string1` and `string2`, so I added a separating space.